test/WebJobs.Script.Tests.Shared/TestConfigurationBuilderExtensions.cs

```diff
@@ -3,16 +3,31 @@
 
 using System;
 using System.IO;
+using System.Linq;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
 
 namespace Microsoft.Azure.WebJobs.Script.Tests
 {
     public static class TestConfigurationBuilderExtensions
     {
-        public static IConfigurationBuilder AddTestSettings(this IConfigurationBuilder builder)
+        private const string ConfigFile = "appsettings.tests.json";
```

Comment:
With my suggestion above, changes to this file are likely not needed

Reply:
Indeed they're not. Removed.